src/core/trulens/instrument/custom.py

```diff
@@ -0,0 +1,543 @@
+"""Custom class application
```

Comment:
Did this change get pulled in from another PR?

Reply:
oh yea looks like it